runelite-api/src/main/java/net/runelite/api/MenuAction.java

```diff
@@ -135,6 +135,13 @@
 	 */
 	USE(38),
 	
+	/**
+	 * Menu action triggered when interacting with a widget's menu option.
+	 * E.g: opening world map, enabling/disabling xp drops, setting up
+	 * quick-prayers, switching tab (Stats, Inventory, etc..).
+	 */
+	WIDGET_OPTIONS(57),
```

Comment:
This looks unused, why is it here?

Reply:
It is needed to allow Add friend, Add ignore, Report, Message and Delete to function as normal. If you would like to complete the MenuAction enum in the future, you can take a look at a complete list here: https://github.com/OSHD/hdhelperclient/blob/master/client/src/main/java/com/hdhelper/client/api/action/ActionTypes.java